spec/shared/hutch_notifications.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+shared_context "hutch notifications", shared_context: :metadata do
+  before { Hutch.connect }
+  after { Hutch.disconnect }
+
+  it "publish the notification" do
+    notification = try(:publish_notification)
+
+    if notification
+      expect(Hutch.broker.exchange).to receive(:publish).once.with(
```

Comment:
Why depending on `hutch` internals? Can we assert the message passed to `Hutch` directly?

Reply:
you are right, I've copied it from Hutch tests